api/clients/v2/examples/client_construction.go

```diff
@@ -28,18 +28,13 @@ import (
 // These constants are specific to the EigenDA holesky testnet. To execute the provided examples on a different
 // network, you will need to set these constants to the correct values, based on the chosen network.
 const (
-	ethRPCURL         = "https://ethereum-holesky-rpc.publicnode.com"
-	disperserHostname = "disperser-testnet-holesky.eigenda.xyz"
-
-	// Eth client
-	eigendaDirectoryAddress = "0x90776Ea0E99E4c38aA1Efe575a61B3E40160A2FE"
-
-	// Router address provider
-	certVerifierRouterAddress = "0x7F40A8e1B62aa1c8Afed23f6E8bAe0D340A4BC4e"
-
-	// Cert builder
+	ethRPCURL                        = "https://ethereum-holesky-rpc.publicnode.com"
+	disperserHostname                = "disperser-testnet-holesky.eigenda.xyz"
+	certVerifierRouterAddress        = "0x7F40A8e1B62aa1c8Afed23f6E8bAe0D340A4BC4e"
 	registryCoordinatorAddress       = "0x53012C69A189cfA2D9d29eb6F19B32e0A2EA3490"
 	blsOperatorStateRetrieverAddress = "0x003497Dd77E5B73C40e8aCbB562C8bb0410320E7"
+	eigenDAServiceManagerAddress     = "0xD4A7E1Bd8015057293f0D0A557088c286942e84b"
```

Comment:
nit: examples should use SOTA stuff, so should only use the eigendaDirectoryAddress.

Reply:
`blsOperatorStateRetrieverAddress` is still used for CertBuilder, that's not changed within this PR. 
I will update `eigenDAServiceManagerAddress` to an empty string because new EthClient take all three arguments